### PR TITLE
Remove unicode/("u")

### PR DESCRIPTION
**Pull request by jona-samuel**
working on #2128

### CODE REVIEW

File: privacyidea/lib/token.py [hunk 1]
```diff
@@ -2449,15 +2449,15 @@ def get_dynamic_policy_definitions(scope=None):
     for ttype in get_token_types():
         pol[SCOPE.ADMIN]["enroll{0!s}".format(ttype.upper())] \
             = {'type': 'bool',
-               'desc': _(u"Admin is allowed to initalize {0!s} tokens.").format(ttype.upper()),
+               'desc': _("Admin is allowed to initalize {0!s} tokens.").format(ttype.upper()),
```
[the remark this answers is not in the document]
Comment (NNLemling) [reply privacyidea/lib/token.py:2452]: While we're at it, please fix the spelling mistake

**Comment by NNLemling** [comment]
Please check the following files:

privacyidea/tools/privacyidea-create-sqlidresolver-user         **line:45:**       
privacyidea/tools/privacyidea-create-pwidresolver-user         **line:45:**

**Review by Sebastian-Lemling — CHANGES_REQUESTED** [review]
Please check the following files:

privacyidea/tools/privacyidea-create-sqlidresolver-user line:45:
privacyidea/tools/privacyidea-create-pwidresolver-user line:45:

File: privacyidea/lib/token.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jona-samuel) [reply privacyidea/lib/token.py]: https://github.com/privacyidea/privacyidea/pull/3474/commits/3e03fb02c7ae87ee14200346b20c52b9e250155b

**Review by Sebastian-Lemling — APPROVED** [review]

**Pull request merged**